mapmvt.c

```diff
@@ -544,7 +544,7 @@ int msMVTWriteTile( mapObj *map, int sendheaders ) {
   if( sendheaders ) {
     msIO_fprintf( stdout,
 		  "Content-Length: %d\r\n"
-		  "Content-Type: application/x-protobuf\r\n\r\n",
+		  "Content-Type: application/vnd.mapbox-vector-tile\r\n\r\n",
```

Comment:
instead of hardcoding the value, we could use MS_IMAGE_MIME_TYPE(map->outputformat)

Reply:
Fixed, thanks